### ISSUE TITLE
tensorflow2_quickstart-for-beginners

### ISSUE THREAD

**Issue by tongweizj** [tongweizj/experiments_Machine-Learning#7]
https://www.tensorflow.org/tutorials/quickstart/beginner

**Comment by tongweizj** [comment]
[Sequential](https://www.google.com/url?q=https%3A%2F%2Fwww.tensorflow.org%2Fguide%2Fkeras%2Fsequential_model) is useful for stacking layers where each layer has one input [tensor](https://www.google.com/url?q=https%3A%2F%2Fwww.tensorflow.org%2Fguide%2Ftensor) and one output tensor. Layers are functions with a known mathematical structure that can be reused and have trainable variables. Most TensorFlow models are composed of layers. This model uses the [Flatten](https://www.google.com/url?q=https%3A%2F%2Fwww.tensorflow.org%2Fapi_docs%2Fpython%2Ftf%2Fkeras%2Flayers%2FFlatten), [Dense](https://www.google.com/url?q=https%3A%2F%2Fwww.tensorflow.org%2Fapi_docs%2Fpython%2Ftf%2Fkeras%2Flayers%2FDense), and [Dropout](https://www.google.com/url?q=https%3A%2F%2Fwww.tensorflow.org%2Fapi_docs%2Fpython%2Ftf%2Fkeras%2Flayers%2FDropout) layers.

**Comment by tongweizj** [comment]
文档非常清晰的介绍了， 使用tensorflow2构建模型时， 需要做哪些工作！
1. load dataset
2. Build a machine learning model
3. Train and evaluate your model

### load dataset
核心是，
1. 将dataset ，意见什么比例分割到train， test

### Build a ml model
步骤
1. 设计 model
2. 编译 model

###  Train and evaluate your model

4. 训练model

**Comment by tongweizj** [comment]
这一节，
最重要的更新是，介绍了如果用已有模型，做预测！
这个在之前的脚本中没有介绍过
